appscale/tools/appscale.py

```diff
@@ -648,9 +651,12 @@ def tail(self, node, file_regex):
     else:
       keyname = "appscale"
 
+    nodes_json_raw = ""
     try:
       with open(self.get_locations_json_file(keyname)) as f:
-        nodes_json_raw = f.read()
+        json_local_nodes = json.loads(f.read()).get('node_info')
+        if json_local_nodes:
+          nodes_json_raw = json.dumps(json_local_nodes)
```

Comment:
Instead of loading, dumping, and reloading, you can just assign to `nodes` directly and remove the assignment below.